build/node.sh

```diff
@@ -44,6 +44,7 @@ else
 fi
 
 echo ""
+echo "RECOMMENDED: inspect the generated configuration ./run_data/conduit.yml"
```

Comment:
The previous line mentions the configuration file (`$CONFIG_FILE`), I'm not sure this is enough and/or any clearer.

Reply:
True. Maybe this should be added to the README instead. Or make this into step 1. So that the output looks like:

```sh
❯ make run
cd cmd/conduit && go build -ldflags='-X github.com/algorand/conduit/version.Hash=1bf75ec8776f7fa5b522c035c161d936cd91ff62 -X github.com/algorand/conduit/version.ShortHash=1bf75ec -X github.com/algorand/conduit/version.CompileTime=2023-06-20T15:05:20+0000 -X "github.com/algorand/conduit/version.ReleaseVersion=Custom Plugin Build"'
./cmd/conduit/conduit -v
Conduit Custom Plugin Build (1bf75ec)
./build/node.sh

Configuration initialized at run_data/conduit.yml.

1. Inspect/modify the generated configuration ./run_data/conduit.yml to customize plugin configuration
2. Start a node with docker: make docker-node
3. Run conduit: ./cmd/conduit/conduit -d run_data
   Optionally add -r 28000000 to start on a specific round.
```

Reply:
I think you're onto something, but maybe all of this should be in a markdown file instead of a script.